index.js

```diff
@@ -256,12 +256,18 @@ function chunkLength (chunk, encoding) {
 
 function shouldCompress (req, res) {
   var type = res.getHeader('Content-Type')
+  var noCompressionHeader = res.getHeader('x-no-compression')
 
   if (type === undefined || !compressible(type)) {
```

Comment:
`compressible` is being called even if added `noCompressionHeader` is present. `noCompressionHeader` should be checked first.

Reply:
good catch